src/nft/components/collection/CollectionAsset.tsx

```diff
@@ -122,7 +122,7 @@ export const CollectionAsset = ({
       removeAssetFromBag={handleRemoveAssetFromBag}
     >
       <Card.ImageContainer isDisabled={asset.notForSale}>
-        <StyledContainer>
+        <StyledContainer data-testid="nft-collection-asset">
```

Comment:
is there no issue with having multiple assets with the same id? should we make them unique based off tokenId?

Reply:
I can't reliably predict the nft assts that will show up.  I just want to verify they render in general

Reply:
okay, this is just a nit but can it by `data-testClass` or something more generic than id?